README.md

```diff
@@ -1,358 +1,45 @@
-        host: "localhost:6379",
-        password: "",
-        options: { connectionPooling: true, isClusterConnection: false, ssl: false,
-            startTls: false, verifyPeer: false, connectionTimeout: 500 }
-};
-
-public function main() returns error? {
-    redis:Client conn =  check new (redisConfig);
-
-    io:println("Pinging Redis Server...");
-    //Ping Server
-    var result = conn->ping();
-    if (result is string) {
-        io:println(result);
-    } else {
-        io:println(result.message());
-    }
-
-    io:println("===Executing sample string oprerations===");
-    //Sample String Operations
-    io:println("Setting value of the key \"Project\" as \"Ballerina\"");
-    var stringSetresult = conn->set("Project", "Ballerina");
-
-    if (stringSetresult is string) {
-        io:println("Reply from the server: " + stringSetresult);
-    } else {
-        io:println("Error occurred while calling `set`");
-    }
-
-    io:println("\nQuerying the server for the value of the key \"Project\"");
-    var value = conn->get("Project");
-
-    if (value is string) {
-        io:println("Reply from the server: " + value);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `get`");
-    }
-
-    io:println("\n===Executing sample list oprerations===");
-    //Sample List Operations
-    io:println("Pushing 2 elements to NumberList");
-    var listPushresult = conn->lPush("NumberList", ["One", "Two"]);
-
-    if (listPushresult is int) {
-        io:println(io:sprintf("Number of elements pushed: %s", listPushresult));
-    } else {
-        io:println("Error occurred while calling `lPush`");
-    }
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-    io:println("Poping an element from NumberList");
-    var lPopResult = conn->lPop("NumberList");
+[Redis](https://redis.io/) is an open source, in-memory data structure store that you can use as a database, cache, and message broker. It supports data structures such as strings, hashes, lists, sets and sorted sets.
 
-    if (lPopResult is string) {
-        io:println("Popped element: " + lPopResult);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `lPop`");
-    }
-    io:println("===Executing sample set oprerations===");
-    //Sample Set Operations
-    io:println("Adding 3 elements to NumberSet");
-    var setAddResult = conn->sAdd("NumberSet", ["1", "2", "3"]);
+The Redis [Ballerina](https://ballerina.io/) connector provides operations to connect and interact with the Redis server over the network.
 
-    if (setAddResult is int) {
-        io:println(io:sprintf("Number of elements added: : %s", setAddResult));
-    } else {
-        io:println("Error occurred while calling `sAdd`");
-    }
+For more information, go to the module(s).
```

Comment:
Suggestion: "For more information about configurations and operations, click the relevant module in [Ballerina Documentation - ballerinax/redis](https://lib.ballerina.io/ballerinax/redis/latest)."

Suggesting this revision where we link to the connector page instead of module pages because it seems like you can't point to the "latest" doc when you directly link to the module docs. Is there a way?

Also, if you add more modules to a connector, updating this documentation can be easily missed.

Reply:
Thing is, then all docs are pointed to the latest version. Before publish, heard to find the link as well. Futher, this link may change if we move Ballerina central to a different domain. As we want to point this README.md to module.md of `this` version, best thing is to use a relative path to module.md